addons/l10n_it_edi_sdicoop/models/account_invoice.py

```diff
@@ -22,3 +22,9 @@ def send_pec_mail(self):
         # With SdiCoop web-service, no need to send PEC mail.
         # Set the state to 'other' because the invoice should not be managed par l10n_it_edi.
         self.l10n_it_send_state = 'other'
+
+    @api.depends('restrict_mode_hash_table', 'state')
```

Comment:
```suggestion
    @api.depends('restrict_mode_hash_table', 'state', 'l10n_it_edi_transaction')
```
Is something like this needed?